Library/Homebrew/cmd/bundle.rb

```diff
@@ -61,42 +61,45 @@ class Bundle < AbstractCommand
 
           `brew bundle env`:
           Print the environment variables that would be set in a `brew bundle exec` environment.
+
+          If `--new-system` is passed or the environment variable `HOMEBREW_USE_NEW_SUBCOMMAND_SYSTEM` is set,
+          use the new subcommand system implementation.
         EOS
         flag "--file=",
              description: "Read from or write to the `Brewfile` from this location. " \
-                          "Use `--file=-` to pipe to stdin/stdout."
+               "Use `--file=-` to pipe to stdin/stdout."
```

Comment:
Don't change whitespace unnecessarily here.

Reply:
reverted. IDE misbehaving for auto indenting and linting